rpcs3/Crypto/unzip.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+std::vector<u8> unzip(const u8* gz, usz size);
+std::vector<u8> unzip(const std::vector<u8>& gz);
```

Comment:
I think this overload should accept anything that the concept `requires (const T& obj) { std::data(obj)[0]; std::size(obj); }` is accepted for.
This way it would be able to accept std::span, string_view, string etc.

Reply:
It does now